org.adempiere.base/src/org/compiere/model/MAttachmentEntry.java

```diff
@@ -24,6 +24,7 @@
 import java.util.Random;
 import java.util.logging.Level;
 
+import org.bouncycastle.util.Arrays;
```

Comment:
Hi @hengsin - is this OK or is it a typo?
Maybe you wanted to use the java.util.Arrays instead of introducing bouncycastle dependency?

Reply:
good catch. wrong import.